doc/classes/EditorSettings.xml

```diff
@@ -772,7 +773,7 @@
 			Translations are provided by the community. If you spot a mistake, [url=$DOCS_URL/contributing/documentation/editor_and_docs_localization.html]contribute to editor translations on Weblate![/url]
 		</member>
 		<member name="interface/editor/editor_screen" type="int" setter="" getter="">
-			The preferred monitor to display the editor.
+			The preferred monitor to display the editor. If set to [code]Auto[/code], the editor will remember the last screen it was displayed on across restarts.
```

Comment:
The actual value of the setting is `-5`, so "Auto" shouldn't be in code tags.
```suggestion
			The preferred monitor to display the editor. If set to [b]Auto[/b], the editor will remember the last screen it was displayed on across restarts.
```
We could also note down the `-5` , but there's no precedent of this _just yet_ so I'd avoid it for consistency.

Reply:
Changing the description! 
> We could also note down the -5 , but there's no precedent of this just yet so I'd avoid it for consistency.
What do you mean? Noting the -5 value in the docs here, or somewhere else?

Reply:
I mean not doing it at all, because this kind of thing is not done almost anywhere else in the class reference.